docs/cache/develop/integrations/sequelize.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 6
+sidebar_label: Sequelize
+title: Integrate Momento With Sequelize ORM
+description: Supercharge your sequelize backed databases with Momento cache
+---
+
+# Integrating Momento with your Sequelize backed database
+
+[Sequelize](https://sequelize.org/) is a popular ORM for Oracle, Postgres, MySQL, MariaDB, SQLite and SQL Server, and more.
+
+In this tutorial, we'll walk through our [Momento sequelize model cache wrapper](https://github.com/momentohq/momento-sequelize-cache). By the end of the exercise, you'll be able to seamlessly cache your sequelize-backed ORM databases with minimal code changes.
+
+## Prerequisites
+
+To get this example running, you'll need to have the following available:
```

Comment:
I like how you re-organized the content so that the example comes later, after your demo code for using in their own app. So I think that at this point in the doc it is confusing to mention the example. I'd probably just say "To use Momento with Sequelize in your app" or something.

Reply:
Yes 100% missed while refactoring